src/client.coffee

```diff
@@ -221,7 +221,7 @@ class Client extends EventEmitter
                 @emit 'new_user', message
             else
                 # Check for `pong` response
-                if message.data?.text? and message.data.text == "pong"
+                if message.data.text? and message.data.text == "pong"
```

Comment:
@loafoe Would you prefer to just close this pull request and I'll submit a new one or is reverting it in the changeset fine?